operator/gitops/argocd/pipeline-service/openshift-pipelines/tekton-config.yaml

```diff
@@ -53,6 +86,36 @@ spec:
     enable-tekton-oci-bundles: true
     options:
       disabled: false
+      configMaps:
```

Comment:
pretty sure this options override was available in 1.13 levels of the operator, but if you could confirm that, I'm fine using your PR vs. mine

only asking just in case another unexpected delay for using nightlies in prod comes up

Reply:
Per http://dashboard.apps.cicd.ospqa.com/product/componentmatrix/ 1.13 is at 0.69.x of the operator

Per https://github.com/tektoncd/operator/blob/release-v0.69.x/pkg/apis/operator/v1alpha1/additional_options.go `AdditionalOptions` allows for the overrides of configmaps

So this is OK for 1.13